tests/commands/test_runner.py

```diff
@@ -86,16 +86,16 @@ def test_integration(self):
 
     def test_debug_enabling(self):
         """
-        {DD,DATADOG}_TRACE_DEBUG=true allows setting debug logging of the global tracer
+        DD_TRACE_DEBUG=true allows setting debug logging of the global tracer.
         """
-        with self.override_env(dict(DATADOG_TRACE_DEBUG="false")):
+        with self.override_env(dict(DD_TRACE_DEBUG="false")):
```

Comment:
we might want to keep a test case with `DATADOG_TRACE_DEBUG` for backwards compatibility checking

Reply:
> we also want to update our corp docs as well: https://github.com/DataDog/documentation/search?q=DATADOG_TRACE_DEBUG

I opened this pr: https://github.com/DataDog/documentation/pull/11152